io.ossia.score.yml

```diff
@@ -0,0 +1,333 @@
+id: io.ossia.score
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: ossia-score
+finish-args:
+  # BLE support
+  - --allow=bluetooth
+
+  # Access user's data and media libraries, plug-ins etc
+  - --filesystem=host
+
+  # GPU rendering support
+  - --device=dri
+
+  # Raw input support
+  - --device=input
+
+  # Required for USB & MIDI devices support as well as JACK (/dev/shm) and raw GPIO I/O
+  - --device=all
+
+  # Removable drives support
+  # https://github.com/flatpak/flatpak/issues/2713#issuecomment-1958510347
+  - --filesystem=/media
+  - --filesystem=/run/media
+
+  # X11 is used by default
+  - --socket=x11
+
+  # Wayland still has rendering regressions compared to X11 as of Qt 6.7
+  - --socket=wayland
```

Comment:
it is either or. Not both. If wayland is broken, don't add it.

Reply:
also VST and VST3 UI support will never work with Wayland so it's moot.

Reply:
okay, I could see many manifests which had both. How would an application handle running on either through QT_QPA_PLATFORM ? (+ also running directly on eglfs without X11 or Wayland involved - the app supports that and it's an important feature when running on embedded boards for art installations, etc.)